examples/annotation_import/tiled.ipynb

```diff
@@ -111,7 +111,7 @@
     {
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #6.      global_keys=global_key, # Paginated collection of data row objects
</pre><p><code>global_keys=[global_key]</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Labelbox/labelbox-python/pull/1006/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>